task-11/Controller/HookNode.php

```diff
@@ -0,0 +1,27 @@
+<?php
+namespace Controller;
+
+require_once("Dao/BrokerNodeDAO.php");
+
+use Dao\BrokerNodeDAO;
+
+class HookNode {
+
+	private $brokerNodeDao;
+
+	function __construct() {
+		$this->brokerNodeDao = new BrokerNodeDAO();
+	}
+	
+	
+	/**
+	* Function to verify with a given IpAddress exists
+	* @author Arthur Mastropietro
+	*/
+	
+	public function verifyRegisteredBrokerNode($ipAddress) {
+		return $this->brokerNodeDao->verifyRegisteredBrokerNode($ipAddress);
+	}
+	
+}
+
```

Comment:
This file seems like it's still expecting there to be a database, rather than a static array of known hooknodes.  We can't do require_once("Dao/BrokerNodeDAO.php"); if that file is gone.